components/core/src/package/install.rs

```diff
@@ -41,6 +41,14 @@ pub struct PackageInstall {
     pub installed_path: PathBuf,
 }
 
+// The docs recommend implementing `From` instead, but that feels a
+// bit odd here.
```

Comment:
Yep, I thought so too since we can't "inflate" a `PackageInstall` from a `PackageIdent`. Seems like implementing `Into` is right in this case.